src/mame/drivers/galaxold.cpp

```diff
@@ -1444,6 +1444,24 @@ static INPUT_PORTS_START( trvchlng )
 INPUT_PORTS_END
 
 
+static INPUT_PORTS_START( bullsdrtg )
+	PORT_INCLUDE( racknrol )
+
+	PORT_MODIFY("IN1")
+	PORT_DIPNAME( 0x40, 0x00, DEF_STR( Cabinet ) )
+	PORT_DIPSETTING(    0x00, DEF_STR( Upright ) )
+	PORT_DIPSETTING(    0x40, DEF_STR( Cocktail ) )
```

Comment:
This is probably a sense line on the wiring harness.  It should be a configuration setting rather than a DIP switch.

Reply:
Thanks.  Change submitted